.github/action/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM node:12-alpine
+# The action uses the last released version of releasify
+RUN npm install releasify -g && releasify -v
```

Comment:
why installing releasify as we already have it as we are in the repo? why not just `COPY . ./` ? I mean also you can target the version if we use proper tagging.